static/src/javascripts/projects/common/modules/crosswords/crossword.js

```diff
@@ -371,9 +371,14 @@ class Crossword extends Component<*, CrosswordState> {
     returnPosition: ?number;
 
     insertCharacter(character: string): void {
+        const characterUppercase = character.toUpperCase();
         const cell = this.state.cellInFocus;
-        if (/[A-Z]/.test(character) && character.length === 1 && cell) {
-            this.setCellValue(cell.x, cell.y, character);
+        if (
+            /[A-Za-zÀ-ÿ]/.test(characterUppercase) &&
```

Comment:
Thanks for this!

It looks like this range won’t allow e.g. Ą or Ș. How about [extending it to cover those too](https://stackoverflow.com/a/39134560)?

Reply:
The unicode range proposed includes some math operators which we'd want to avoid? I hope our users aren't forced to insert obscure alt-codes into crosswords 😅

Reply:
> The unicode range proposed includes some math operators

Can’t see any. And even if, I would rather have an ability to enter some of the obscure glyphs (what’s the danger here?) than not being able to spell Wałęsa. Or [Erdoğan](https://github.com/guardian/frontend/pull/9411).

As a minor point: do you think it would be easy to avoid crossword code preventing hold-key-for-diacritic Mac OS feature from working?
![image](https://user-images.githubusercontent.com/6032869/41589075-7c6ce64a-73ab-11e8-82c9-9c2c166296ff.png)

Again: thanks for a quick fix!

Reply:
I'm not sure. I the keyboard feature doesn't seem to work on my version of OS X El Capitan. 

We could add this to the backlog for someone to investigate.